draft-ietf-lamps-pq-composite-kem.md

```diff
@@ -550,11 +550,13 @@ Encap Process:
 
   4. Encode the ciphertext
 
-     ct = mlkemCT || tradCT
+      ct = mlkemCT || tradCT
 
   5. Combine the KEM secrets and additional context to yield the composite shared secret
-
-     ss = KDF(mlkemSS || tradSS || tradCT || tradPK || Domain)
+      if KDF is "SHA3-256"
+        ss = SHA3-256(mlkemSS || tradSS || tradCT || tradPK || Domain)
+      else if KDF is "HKDF-SHA256"
+        ss = HKDF-SHA256(salt=0x0000000000000000, IKM=mlkemSS || tradSS || tradCT || tradPK || Domain)
```

Comment:
Is that clear enough from parameter names, or should that explicitly be HKDF-Extract-SHA256?

There are two HKDF operations (HKDF-Extract and HKDF-Expand), and parameters being salt and IKM only makes sense for HKDF-Extract.

Reply:
I've been debating with myself whether this should be `HKDF-Extract()`, or `HKDF-Expand( HDKF-Extract() )`.

In the end, I think there is no need for stretching, so only HKDF-Extract() is needed, and that makes things simpler.